app/javascript/schools/courses/components/students_editor/StudentsEditor__TeamsList.res

```diff
@@ -284,12 +303,17 @@ let make = (
   ~updateFilterCB,
   ~refreshTeams,
 ) => {
+  let (count, setCount) = React.useState(() => None)
```

Comment:
I think it's better to store the `totalCount` along with the list in the parent state. 
Also, use `0` as the initial value for the totalCount